sessions/backend-update-delete/demo-end/components/List/index.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+const StyledList = styled.ul`
+  list-style-type: none;
+  display: grid;
+  gap: 1rem;
+  justify-items: center;
+  padding: 0;
+`;
+
+export default function List({ children }) {
+  return <StyledList>{children}</StyledList>;
+}
```

Comment:
This component can be omitted. You can export the styledList directly.

Reply:
Yes, right 🧹